src/Module.php

```diff
@@ -55,4 +57,13 @@ public function getConfig()
     {
         return include __DIR__ . '/../config/module.config.php';
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getModuleDependencies() {
```

Comment:
here is a little cs problem

the { must be in a new line.

than all is fine

Reply:
Fixed the codestyle issue